game/core/view_strategies/rpg_movement.py

```diff
@@ -144,12 +151,10 @@ def on_key_press(self, key, modifiers) -> None:
         #     self.window.show_view(self.window.views["inventory"])
         # elif key == arcade.key.ESCAPE:
         #     self.window.show_view(self.window.views["main_menu"])
-        # elif key in constants.SEARCH:
-        #     self.search()
-        # elif key == arcade.key.KEY_1:
-        #     self.selected_item = 1
-        #     # Will add this to the other keys once there are more items
-        #     player_sprite.equip(1)
+        elif key == arcade.key.KEY_1:
+            self.selected_item = 1
+            # Will add this to the other keys once there are more items
+            self.player_sprite.equip(1)
         # elif key == arcade.key.KEY_2:
         #     self.selected_item = 2
```

Comment:
For when we need it, maybe this code could be uncommented and then instead of calling equip on each, you could do something like:

```py
...
elif key == arcade.key.KEY_9:
    self.selected_item = 9
elif key == arcade.key.KEY_0:
    self.selected_item = 10
if self.selected_item:
    # Will add this to the other keys once there are more items
    self.player_sprite.equip(self.selected_item)
```

Reply:
`selected_item` is for highlighting the selected item briefly on mouse press, so it becomes `None` on mouse release, so we wouldn't be able to use it for the equip. However, as you pointed out in the other comment, it's not working properly, so I just fixed it! 

I also renamed it to `number_pressed` to make it less confusing and moved it all to `game_gui` since no other file really needs to know about it.